Documentation/gettingstarted/hubble_cli.rst

```diff
@@ -25,6 +25,10 @@ across the globe, there is almost always someone available to help.
    unsure, run ``cilium status`` and validate that Cilium and Hubble are up and
    running.
 
+.. note::
+
+    Before you begin, make sure you have deployed the Demo Application together with
+    its network policies. Please refer to :ref:`gs_http_sw_demo` for more details.
```

Comment:
Can we phrase this differently please? More “the examples below use the Demo App presented at...” than “Make sure to deploy this”. This tutorial provides examples on how to use the command line, but it's not a step-by-step tutorial to unroll some scenario-based use case, so I think we shouldn't _push_ the reader to install it, just let them know (Sorry, I'm picky :) ).

Also I'd just link to `_gs_http`, not sure it's worth creating a new target below the requirements. But I don't mind much, so this is your call.

Reply:
> (Sorry, I'm picky :)

hahaha, no worries, this type of thoughtful discussions is what makes open source great 👍

Reply:
one thing though, you do need to have the Demo App deployed to run these examples right ? So, maybe one can say: 
``` 
    This guide uses examples based on the Demo App. If you would like to run them,
    you would need to deploy the Demo App first. Please refer to :ref:`gs_http`
    for more details.
```

Reply:
I pushed something based on this previous comment, I'm always open to discuss if you still think the wording doesn't match the intent. Cheers !

Reply:
Looks good to me, thank you for the change!